### PR TITLE
Fix token accounts to align with latest APIs

### PR DESCRIPTION
**Pull request by awrichar**
FireFly_Explorer

FireFly_Explorer

### CODE REVIEW

File: src/modules/tokens/views/Accounts/AccountDetails.tsx [hunk 1]
```diff
@@ -241,7 +247,7 @@ export const AccountDetails: () => JSX.Element = () => {
               {t('accounts')}
             </Link>
             <Link underline="none" color="text.primary">
-              {t('accountDetails')}
+              {key}
```
Comment (eberger727) [inline src/modules/tokens/views/Accounts/AccountDetails.tsx:250]: Maybe use the HashPopover here and make it a short hash

File: src/modules/tokens/views/Accounts/AccountDetails.tsx [hunk 2]
```diff
@@ -315,6 +290,89 @@ export const AccountDetails: () => JSX.Element = () => {
...
+
+const AccountBox = (options: AccountBoxOptions): JSX.Element => {
+  const { t } = useTokensTranslation();
+  const classes = useStyles();
+  const { balance } = options;
+  const joinedIndexes = balance.tokenIndexes.join(', ');
+  const detailsData = [
+    {
+      label: t('poolID'),
+      value: <HashPopover address={balance.pool}></HashPopover>,
```
Comment (eberger727) [inline src/modules/tokens/views/Accounts/AccountDetails.tsx:330]: use shortHash={true} prop

**Comment by shorsher** [comment]
Just one unused interface error that's failing the build.

**Review by shorsher — APPROVED** [review]
Looks good, leaving it up to if you want to include David's comments

🚀

**Comment by awrichar** [comment]
Fixed the lint error in my previous commit, and added one new commit.

**Comment by awrichar** [comment]
This is the updated Accounts listing with extra pool info pulled in.

FireFly_Explorer
te

**Comment by awrichar** [comment]
@eberger727 there will be a follow-up review to overhaul AccountDetails further. I'll incorporate changes to the hash displays there.

**Pull request merged**